Property_map/include/CGAL/property_map.h

```diff
@@ -39,6 +39,31 @@ namespace CGAL {
 
 /// \cond SKIP_DOXYGEN
 
+  template <typename K, typename V>
+class static_property_map
+{
+  typedef K key_type;
+  typedef V value_type;
+  typedef const V& reference;
+  typedef boost::read_write_property_map_tag category;
+  V v;
+
+public:
+  static_property_map(V pv)
+    :v(pv){}
+  inline friend
+  value_type
+  get(const static_property_map& pm, const key_type&)
+  {
+    return pm.v;
+  }
+
+  inline friend
+  void
+  put(static_property_map& pm, const key_type&, const value_type&)
+  {}
+};
```

Comment:
@afabri:

- That class `static_property_map` should be named `Static_property_map`, to follow CGAL naming conventions,
- how is that different from the better implementation `boost::static_property_map`?